packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/users/resetPassword/ResetPassword.modal.tsx

```diff
@@ -30,6 +37,10 @@ const ResetUserPassword = () => {
   const user = users?.find((u) => u.id === userId);
 
   const [newPass, setNewPass] = useState<string>();
+  const [selectedEndpoint, setSelectedEndpoint] = useState<
+    database.service.Endpoint
+  >(service.endpoints[0]);
+
```

Comment:
Regarding my previous comment, should look like this :)

```suggestion
  const [selectedEndpoint, setSelectedEndpoint] = useState<
    database.service.Endpoint | undefined
  >(service.endpoints[0]);

```

Reply:
Ah yes, my bad. Done :)